included/9.5.0/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 9.5.0 cypress/browsers:node16.5.0-chrome97-ff96
+#
+# build this image with command
+#   docker build -t cypress/included:9.5.0 .
+#
+FROM cypress/browsers:node16.5.0-chrome97-ff96
```

Comment:
Images that are created with a node version < `16.10` will not work with the most recent version of Angular (13). We keep getting issues filed as a result of this

Reply:
We should probably generate a new browsers image with at least that version of node as a first step - I pulled the most recent one from the list without much thought. https://github.com/cypress-io/cypress-docker-images/tree/master/browsers

Reply:
Great Idea. I created https://github.com/cypress-io/cypress-docker-images/issues/598 to track this work